SecretManagement.KeePass.Extension/Public/Get-SecretInfo.ps1

```diff
@@ -1,30 +1,74 @@
+using namespace Microsoft.PowerShell.SecretManagement
 function Get-SecretInfo {
     param(
         [string]$Filter,
         [string]$VaultName = (Get-SecretVault).VaultName,
-        [hashtable]$AdditionalParameters = (Get-SecretVault -Name $VaultName).VaultParameters
+        [hashtable]$AdditionalParameters = (Get-SecretVault -Name $VaultName).VaultParameters,
+        [Switch]$AsKPPSObject
     )
     if (-not (Test-SecretVault -VaultName $vaultName)) {throw "Vault ${VaultName}: Not a valid vault configuration"}
 
     $KeepassParams = GetKeepassParams -VaultName $VaultName -AdditionalParameters $AdditionalParameters
-    $KeepassGetResult = Get-KPEntry @KeepassParams | 
-        ConvertTo-KPPSObject |
-        Where-Object {$_ -notmatch '^.+?/Recycle Bin/'}
+    $KeepassGetResult = Get-KPEntry @KeepassParams | ConvertTo-KPPSObject
+    if (-not $AdditionalParameters.ShowRecycleBin) {
+        $KeepassGetResult = $KeepassGetResult | Where-Object FullPath -notmatch '^[^/]+?/RecycleBin$'
+    }
+
+    #TODO: Split this off into private function for testing
+    function Get-KPSecretName ([PSCustomObject]$KPPSObject) {
```

Comment:
This is all code that was added in 6aff52851ab1a6ec4fae19749512ed23c9547705, it shouldn't be showing up here. Doesn't seem to cause a merge conflict because it's probably the exact same code

@trir262 I don't see a rebase commit, so I don't think you pushed it (you have to force push a rebase). Don't worry about the rebase for now, we'll discuss it later. I think this is showing up because you merged in one of my WIP PRs, but after the squash merge the commit history changes and is no longer detected as being the same changes.

So no changes here, just FYI